src/app/share/views/SharedLinksView/SharedLinksView.tsx

```diff
@@ -211,6 +211,18 @@ export default function SharedLinksView(): JSX.Element {
       : dispatch(uiActions.setIsShareDialogOpen(true));
   };
 
+  const getDetails = useCallback(
+    (item) => {
+      const itemDetails = {
+        ...item,
+        isShared: true,
+      };
+      dispatch(uiActions.setItemDetailsItem(itemDetails));
+      dispatch(uiActions.setIsItemDetailsDialogOpen(true));
+    },
+    [dispatch, uiActions],
+  );
```

Comment:
This function is repeated with some change, maybe it would be a good idea to try to extract it.